api/apiconsumer/models.py

```diff
@@ -46,3 +47,6 @@ def access_secret(self):
 
     def __unicode__(self):
         return "%s (level %d)" % (self.name, self.permission_level)
+
+    def __str__(self):
+        return str(self.name) + str(self.email) + str(self.description) +  str(self.token) + str(self.is_active)
```

Comment:
Format this with commas or dashes in order to better differentiate the fields before merging.